src/extract_slides.js

```diff
@@ -473,6 +502,39 @@ markdownTokenRules['list_item_open'] = function(token, env) {
 
 markdownTokenRules['list_item_close'] = NO_OP;
 
+// These rules are specific to parsing markdown in an inline context.
+
+inlineTokenRules['heading_open'] = function(token, env) {
+    startStyle({bold: true}, env); // TODO - Better style for inline headers
+};
+inlineTokenRules['heading_close'] = function(token, env) {
+    endStyle(env);
+};
+inlineTokenRules['inline'] = markdownTokenRules['inline'];
```

Comment:
This list of assignments is long enough to warrant some optimization. What about creating a base / shared / common token rules which both markdownTokenRules and inlineTokenRules are copies of, before they add their own custom handlers.